### ISSUE TITLE
Implement a `load more` component that uses events pagination to load events

### ISSUE THREAD

**Issue by apoclyps** [apoclyps/my-dev-space#443]
**Is your feature request related to a problem? Please describe.**
As a User, I would like faster page loads to spend less time waiting for events. However, I would also like to view all the events for the service (not just those that are available on page load). 

**Describe the solution you'd like**
A `Load More` component at the bottom of the page that when activated, loads another page worth of data. e.g. the default page size of 50 events, would trigger another 50 results, and another 50 until there are no results remaining.

**Describe alternatives you've considered**
Open to alternative ideas.

**Additional context**
Follow up to #422

**Comment by alistairjcbrown** [comment]
Would this be better implemented as a piece of middleware that listens to the window scroll and makes the next request when the scroll positions nears the bottom?

**Comment by apoclyps** [comment]
Yes. Good call. I hadn't considering using a middleware for this. Is there any standard or common middleware that do this or is it something that is fairly custom?

**Comment by Wgil** [comment]
As pointed by @alistairjcbrown, we could implement this using [React visibility sensor](https://github.com/joshwnj/react-visibility-sensor). As this site is `100vh` long, we can rely of `containment` prop. Instead of watching for the window viewport, we can listen for the containing component height.

@apoclyps would you like me to give it a try?

**Comment by apoclyps** [comment]
@Wgil that sounds great. I will send you an invitation to collaborate and assign the issue to you.

**Comment by Wgil** [comment]
Is there a way to start the project locally so I can see Events without setting up the python server? I'm having troubles setting up all the environment.

**Comment by apoclyps** [comment]
In the root of your repo, if you change the React app events server environment variable from local host to 'https://my-dev-space-staging-alb-2128504978.us-east-1.elb.amazonaws.com/event' you can bypass setting up a python server.

Note: you may need to do a npm install in the root of the repo, within services/client, and rebuild and bring up docker.

**Comment by Wgil** [comment]
I'm getting a lot of unresolved modules error when I try to start the client.
For example:
`Module not found: Can't resolve 'components/call-to-action-banner' in '/usr/src/app/src/pages/events/components'`
Am I missing something?

**Comment by alistairjcbrown** [comment]
@Wgil Your `NODE_PATH` will need to be set, see the `.env` file  - how are you starting the client?
If you use docker it'll read from `.env` and set the environment up; if you aren't using that (eg. `npm start`), you'll need to set it up yourself.

**Comment by Wgil** [comment]
@alistairjcbrown I'm sorry I messed up that file, now it's working again. I'm starting the client with `npm start`. Now I get a connection refused to this url `https://my-dev-space-staging-alb-2128504978.us-east-1.elb.amazonaws.com/event/events`

**Comment by Wgil** [comment]
@alistairjcbrown it does!

**Comment by apoclyps** [comment]
Sorry, this was a mistake on my part; the URL doesn't need to comtain the `/events`. It should be:

```
REACT_APP_EVENTS_SERVICE_URL=https://my-dev-space-staging-alb-2128504978.us-east-1.elb.amazonaws.com
```

**Comment by Wgil** [comment]
Is pagination currently working? I'm receiving about 260 upcoming events and not 50 as the default.

http://my-dev-space-staging-alb-2128504978.us-east-1.elb.amazonaws.com/events?page=1&page_size=1 doesn't work for me.

**Comment by apoclyps** [comment]
@Wgil Looks like staging is a few commits behind; I'll bump the staging environment now: https://github.com/apoclyps/my-dev-space/commits/staging

**Comment by Wgil** [comment]
@apoclyps it doesn't work here either https://muxer.co.uk/events?page=1&page_size=1

**Comment by apoclyps** [comment]
@Wgil The changes are now live on staging:

http://localhost/events?page=1&page_size=1

<img width="1634" alt="screen shot 2018-10-14 at 20 02 28" src="https://user-images.githubusercontent.com/1443700/46920867-68115d00-cfec-11e8-98d9-847d3429be2d.png">

**Comment by apoclyps** [comment]
Thanks again @Wgil for this PR. Closed by #469
